cmds/chmod/chmod_test.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	testPath = "."
+	// if true removeAll the testPath on the end
+	remove = true
+)
+
+const (
+	maxSizeFile = 4096
```

Comment:
Which of these constants do you still need? You might want to double check before you do the commit.